### PR TITLE
[Blog] week 12

### LINKED ISSUE #117: Week 12
Just to not forget that I moved the packit-service stable branch in the middle of the week (now) to include https://github.com/packit-service/packit-service/pull/493 and https://github.com/packit-service/packit-service/pull/496

EDIT: also https://github.com/packit-service/packit-service/pull/497 and https://github.com/packit-service/packit-service/pull/499

EDIT2: also https://github.com/packit-service/packit-service/pull/498 and https://github.com/packit-service/packit-service/pull/505

### PR DESCRIPTION
**Pull request by lbarcziova**
Fixes #117

### CODE REVIEW

File: content/posts/March-2020.md [hunk 1]
```diff
@@ -70,4 +70,41 @@ draft: false
...
+  you do not have `authentication` key in the configuration [(#754)]
+
+
+
+### packit-service
+* We periodically check the status of the copr-build so that we do not need to rely
+  on the result message, this should avoid failures in reporting the status of build [(#490)]
+* `Propose-update` triggered by comment in Github now recognizes also pre-releases [(#498)]
+* We have updated the targets in Makefile so you can now run requre tests without
+  Openshift [(#497)]
```
Comment (jpopelka) [inline content/posts/March-2020.md]: end users probably don't care about our `Makefile`

File: content/posts/March-2020.md [hunk 2]
```diff
@@ -70,4 +70,41 @@ draft: false
...
+* Testing farm in Packit-service works again, we fixed several bugs
```
Comment (jpopelka) [inline content/posts/March-2020.md]:
```suggestion
Several bugs in how we report results from Testing farm have been fixed
```

File: content/posts/March-2020.md [hunk 3]
```diff
@@ -70,4 +70,41 @@ draft: false
...
+
+## Week 12 (March 16th - March 20th)
+
+
+
+### packit
+* When you are executing `copr-build` command, you do not 
+  need to set the project name if this value is defined in
+  the copr-build job in the configuration file [(#763)]
+* Packit orders all patches that are generated when adding them into a specfile [(#765)]
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply content/posts/March-2020.md]: Yeah, the fix I did doesn't work and is not changing anything in the end.

Let's either write
```
All patches generated from a source-git repo are now prefixed with a number so they are easy to sort.
```

or remove the line. Thanks.

File: content/posts/March-2020.md (hunk 2 above)
Comment (TomasTomecek) [reply content/posts/March-2020.md]: +"...and testing farm results should now appear correctly in commit status checks"

**Pull request merged**